dist/app.css

```diff
@@ -15,3 +12,6 @@
 .DO8HfK3ZS4ejgwpTbHe3:hover {
   color: red;
 }
+.g5jsXY3ijcgYEXcj3zzW {
+  color: blue;
+}
```

Comment:
Even though `.unused { ... }` is not referenced by `Counter.tsx`, it remains in the bundle.